docs/source/usersguide/settings.rst

```diff
@@ -336,6 +336,16 @@ or particles going to a cell::
 .. note:: The ``cell``, ``cellfrom`` and ``cellto`` attributes cannot be
           used simultaneously.
 
+To generate surface source files in different batches, ``batches`` is available. As an 
```

Comment:
Very thorough to include an example here. Thank you!

Reply:
Might be worth noting that the simulation bank is cleared here as well for clarity on the purpose of this feature.

Reply:
Done!